src/clj/game/cards/programs.clj

```diff
@@ -144,6 +144,26 @@
                    :msg (msg "install " (:title target))
                    :effect (req (when (can-pay? state side nil :credit (:cost target))
                                   (runner-install state side target)))}]})
+   "Consume"
+   {:events {:runner-trash {:req (req (= (:side target) "Corp"))
+                              :optional
```

Comment:
This keyword should be aligned with `:req` above. The opening brace below as well, or alternatively on the same line as `:optional`